paasta_tools/setup_kubernetes_cr.py

```diff
@@ -281,10 +277,10 @@ def format_custom_resource(
         "spec": instance_config,
     }
 
-    url = get_dashboard_url(kind, service, instance, cluster)
+    url = get_dashboard_base_url(kind, cluster)
     if url:
-        resource["metadata"]["annotations"]["yelp.com/dashboard_url"] = url
-        resource["metadata"]["annotations"][paasta_prefixed("dashboard_url")] = url
+        resource["metadata"]["annotations"]["yelp.com/dashboard_base_url"] = url
+        resource["metadata"]["annotations"][paasta_prefixed("dashboard_base_url")] = url
```

Comment:
Have you checked with #paasta that they are happy with you changing the name of this common annotation?

Reply:
This annotation is only used for Flink services. The code path that the changes touch is limited to Flink.